### PR TITLE
[IMP] Adyen: Update directions for setting the URLs

### PR DESCRIPTION
**Pull request by auva-odoo**
task-3338125

### CODE REVIEW

**Review by AntoineVDV — CHANGES_REQUESTED** [review]
LGTM, but I think we should also provide the test URLs now and not in another task, as we first discussed. The reason is that, in the previous API URLs page, Adyen listed the test URLs, which allowed users to follow the same configuration steps in both live and test environments.

**Review by zulu-odoo — CHANGES_REQUESTED** [review]
Hellooo,
Just two tiny things, I leave it up to you to decide if they're valid or not :)
Thank youuu

File: content/applications/finance/payment_acquirers/adyen.rst [hunk 1]
```diff
@@ -47,8 +47,10 @@ API Key and Client Key
 
 In order to retrieve the API Key and the Client Key, log into your Adyen account, go to
 :menuselection:`Developers --> API Credentials`.
+
 - If you already have an API user, open it.
 - If you don't have an API user yet, click on **Create new credential**.
+
 Go to :menuselection:`Server settings --> Authentification` and copy or generate your **API Key**.
 Be careful to copy your API key as you'll not be allowed to get it later without generating a new
```
[the remark this answers is not in the document]
Comment (auva-odoo) [reply content/applications/finance/payment_acquirers/adyen.rst:55]: You're right, but I will take care of this in another PR when I review the structure of the whole page (it needs a lot of TLC) :-)

File: content/applications/finance/payment_acquirers/adyen.rst [hunk 2]
```diff
@@ -88,17 +90,30 @@ You have to save the webhook to finalize its creation.
...
-   :align: center
-   :alt: Get the links for the different API.
+#. Log into your Adyen account, then go to :menuselection:`Developers --> API URLs`.
+#. Copy the :guilabel:`Prefix` for your live Customer area (i.e., **data center**) and save it for
+   later.
+
+   .. image:: adyen/adyen-api-urls.png
+     :alt: Copy the prefix for the Adyen APIs
+
+#. In Odoo, :ref:`navigate to the payment acquirer Adyen <payment_acquirers/add_new>`.
```
[the remark this answers is not in the document]
Comment (auva-odoo) [reply content/applications/finance/payment_acquirers/adyen.rst:106]: It's still "payment acquirers" in v15 but I will have to change it for v16 :-)

**Review by zulu-odoo — APPROVED** [review]

**Review by AntoineVDV — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**